pkg/minishift/hostfolder/hostfolder.go

```diff
@@ -321,7 +321,7 @@ func mountCifsHostfolder(driver drivers.Driver, hostfolder *config.HostFolder) e
 	}
 
 	cmd := fmt.Sprintf(
-		"sudo mount -t cifs %s %s -o username=%s,password=%s",
+		"sudo mount -t cifs %s %s -o username=%s,password=%s,context=system_u:object_r:svirt_sandbox_file_t:s0",
```

Comment:
@gbraad Does this also going to work with b2d side?

Reply:
likely the option gets ignored, but this needs to be tested